### DISCUSSION TITLE
Can,t Publish station on plublic directories

### DISCUSSION THREAD

**Discussion by bytension** [AzuraCast/AzuraCast#6237] [Q&A] [answered]
Hi! i have a running a station using a 128kbps mp3 mountpoint at my local server (Docker mode). Everything works ok but can,t get it listed on two exampled icecast directories (internet-radio.com and dir.xiph), I,ve followed instructions i found on  previous discussions:

1-Activated option to plublish station on Yellow Pages (At mount point settings)
2-Added this to the advanced configuration part of the mount point settings :

```
<directory>
                <yp-url-timeout>15</yp-url-timeout>
                <yp-url>http://icecast-yp.internet-radio.com</yp-url>
</directory>
<directory>
                <yp-url-timeout>15</yp-url-timeout>
                <yp-url>http://dir.xiph.org/cgi-bin/yp-cgi</yp-url>
</directory>
```
3- Restarted radio stream

Non of both directories shows my station after long time (days)  Researched here and the web, and can,t find if i,m doing something wrong. Any help or hint?  Thank you!

**Comment by Moonbase59** [comment, marked as answer]
Just tried to reproduce your issue on Rolling Release #[4b05857](https://github.com/AzuraCast/AzuraCast/commit/4b05857755aa380ec829156867eb09f5383a1b52) (2023-04-24 15:58), and it worked in minutes (using only [dir.xiph.org](https://dir.xiph.org/)).

You need to switch your station to "Advanced" mode and add:
```xml
<directory>
    <yp-url-timeout>15</yp-url-timeout>
    <yp-url>http://dir.xiph.org/cgi-bin/yp-cgi</yp-url>
</directory>
```
in _Profile → Edit profile → Broadcastin → Custom Configuration (Advanced)_.

Then, in _Broadcasting → Mount Points → (mount point) → Edit_, activate "Publish to "Yellow Pages" Directories" for each mount point that you wish listed on the yellow pages.

You _will_ need to _restart broadcasting_ after these changes. The Xiph.org directory server should pick up your station within minutes. (As of April, 2023)

I advise to check with dir.xiph.org first, and if that works out, add the _internet-radio.com_ directory as a second step.

**Reply by bytension** [reply]
Followed your steps and now it,s woking. Something wrong at my previous config.

Much thanks for your help. :)

**Reply by Moonbase59** [reply]
Happy to help. Does the playing link work for you? (see https://github.com/AzuraCast/AzuraCast/issues/6241)

**Reply by bytension** [reply]
Got exactly the same issue here (on Xiph.org, but on internet-radio works ok) . After a little investigation,  in my case ,  it,s related to a certificate validation problem.  The assigned port for retranmission (8000) differs from 443 standard for HTTPS. My firewall sends this request  (8000) to the azuracast  server directly instead the reverse proxy (that holds the domain certificate) and the web explorer fails to validate it even you have the port open and exposed to the internet.

I tried  to set on mount point setting the retransmission URL to use the predefined standard URL for stream s but it does not works.

Il move any further info i could get to #6241.

**Reply by Moonbase59** [reply]
Yes, it will then forward port 8000 to AzuraCast which in turn will present its own self-signed certificate, instead of the one the reverse proxy holds. One reason more to be able to set the listener URL within Icecast…
